packages/core/content-manager/server/services/utils/populate.js

```diff
@@ -168,8 +168,7 @@ const getDeepPopulateDraftCount = (uid) => {
 
     switch (attribute.type) {
       case 'relation': {
-        const childModel = strapi.getModel(attribute.target);
-        if (hasDraftAndPublish(childModel) && isVisibleAttribute(model, attributeName)) {
+        if (isVisibleAttribute(model, attributeName)) {
```

Comment:
Have not tried, but I wonder what happens if the relation targets another one inside a component.
Components will not have the published_at attribute right?

I will try later during the day what happens if I try to publish an entry that has a relation targeting a compo.

Reply:
Hum 🤔  you can't have a relation targeting a component

Reply:
oh, we don't ? never mind then :D I thought we supported relations inside components